ui/decredmaterial/collapsible.go

```diff
@@ -6,280 +6,78 @@ import (
 	"gioui.org/layout"
 	"gioui.org/unit"
 	"gioui.org/widget"
-	"gioui.org/widget/material"
 )
 
-type MoreItem struct {
-	Text   string
-	Button *widget.Clickable
-	label  Label
-}
-
 type Collapsible struct {
 	IsExpanded      bool
 	Button          *widget.Clickable
 	BackgroundColor color.RGBA
-	expandedIcon    *widget.Icon
-	collapsedIcon   *widget.Icon
+	expandedIcon    *widget.Image
+	collapsedIcon   *widget.Image
 	card            Card
```

Comment:
Any reason why the collapsible widget was modified? Only changes that needed to be made seems to be from the wallets page